### ISSUE TITLE
Full thermodynamic profile

### ISSUE THREAD

**Issue by slochower** [slochower/paprika-manuscript#2]
> (NMH: Is the "full thermodynamic profile" meant to be distinguishing aspect of APR? because Yank could do that do.  It's more of a relative vs absolute thing I think)

I meant ΔG, ΔH, and -TΔS together.

**Comment by nhenriksen** [comment]
i understood ... the way it's phrased makes it sound as if it is unique to APR, but you can get it with just about any absolute method.

**Comment by slochower** [comment]
Okay, good point. I don't see a good example of how to compete enthalpies with YANK, but maybe that comes for free when you do the regular analysis.

**Comment by nhenriksen** [comment]
right, it might not be built-in, but it's just a matter of recording potential energies from the endpoints.  that said, it is notable if we explicitly support it and make it easy to calculate.
